editor/.dependency-cruiser.js

```diff
@@ -32,6 +32,19 @@ module.exports = {
     //   },
     //   to: {},
     // },
+    {
+      name: 'not-from-monaco-to-editor',
+      comment:
+        'The code editor files should not accidentally import the main editor, as that pulls in 10+ megabytes of dependencies',
+      severity: 'error',
+      from: {
+        path: 'monaco-editor-iframe\\.tsx$',
+      },
+      to: {
+        path: ['src/templates/editor\\.tsx', 'src/components/editor/store/editor-state\\.ts'],
```

Comment:
Probably worth extending the `'src/templates/editor\\.tsx'` to become everything except `src/templates/monaco-editor-iframe.tsx`

Reply:
you mean everyting inside /templates?

Reply:
Ha yeah!